evidence.go

```diff
@@ -78,10 +78,16 @@ func (e *Evidence) FromCOSE(cwt []byte, supportedProfiles ...string) error {
 	// the condition has been checked at the top of the function by the call
 	// to checkSupportedProfiles
 	for _, profile := range supportedProfiles {
-		err = e.Claims.validate(profile)
-		if err == nil {
+		e := e.Claims.validate(profile)
```

Comment:
Where is the err assigned here ?

Reply:
I am not sure I understand which err?  Is it `e` or `err` that you are wondering about?

Reply:
I mean you are checking now err, but where it is getting set, as now there is no err in the assignment but just e ?

Reply:
Discussed with @thomas-fossati and two corrections required:

(a) Rename return variable for validate function call to verr

(b) If verr is nil, then reset the err to nil

Reply:
Done in 377a8de